res/values/strings.xml

```diff
@@ -269,7 +269,7 @@
     <string name="send_failed">Could not send message</string>
 
     <!-- map -->
-    <string name="filter_map_on_time">Show locations in time frame</string>
```

Comment:
i'd say time frame is correct here. timeframe is more sth. used on video-processing.

Reply:
It pains me, but will revert.